framework/src/Volo.Abp.TextTemplating.Abstractions/Volo/Abp/TextTemplating/AbpTemplateRendererProviderOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Volo.Abp.TextTemplating
+{
+    public class AbpTemplateRendererProviderOptions
+    {
+        protected IDictionary<string, Type> Providers { get; }
+
+        public AbpTemplateRendererProviderOptions()
+        {
+            Providers = new Dictionary<string, Type>();
+        }
+
+        public AbpTemplateRendererProviderOptions AddProvider<TProvider>(string templateRenderEngineName)
```

Comment:
We use two terms for the same thing: "render engine" and "renderer provider". I think we should go with one of them. "Rendering engine" seems reasonable to me. So, we should also rename `AbpTemplateRendererProviderOptions` to `AbpTemplateRenderingEngineOptions`, or I even suggest to simply define `AbpTextTemplatingOptions` and use a RenderEngines dictonary inside it.